src/main/java/teammates/common/util/Utils.java

```diff
@@ -33,5 +35,17 @@ public static Gson getTeammatesGson() {
                 .setPrettyPrinting()
                 .create();
     }
-
+    
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
@damithc Hmm this method looks like it should be specifically for a list of the enum type `FeedbackParticipantTypes`, as there is already the method `toString(List<String> strings, String delimiter)` in `StringHelper` that joins a list of strings separated by the specified delimeter.